### DISCUSSION TITLE
23 gennaio 2023 A - esercizio 2

### DISCUSSION THREAD

**Discussion by atyion** [sapienzastudentsnetwork/basi-di-dati-1#21] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/6d0b1953-52e1-4edb-8e17-bc5eec0d6aac)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/f6471707-3ad4-40e9-997c-adbf9e2d6119)

**Comment by niccolo-fato** [comment]
Soluzione analoga a @atyion:
![2024-01-13 16 58 47](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/6c595627-b767-4995-a08b-a200f10dbb64)
![2024-01-13 16 58 57](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/d81b7a75-f0d3-4d70-a4ab-bb4c012c3675)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-07 122203](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/436f162d-d2b3-427e-939c-a0fd53ddd114)
![Immagine 2024-02-07 122213](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/5dc6a1e0-26a3-43ad-86b5-088224318def)
![Immagine 2024-02-07 122235](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/ac4e8f9f-0bfd-4931-9f3a-67d89855a54e)
